.yarn/versions/0d3ddb4d.yml

```diff
@@ -0,0 +1,7 @@
+releases:
+  "@yarnpkg/plugin-essentials": patch
+
+declined:
+  - "@yarnpkg/plugin-interactive-tools"
+  - "@yarnpkg/plugin-typescript"
+  - "@yarnpkg/cli"
```

Comment:
`@yarnpkg/cli` needs to marked for release as well.

Reply:
https://github.com/yarnpkg/berry/pull/4800/commits/65e1261eb9a04b7f725d51a28f0ee9158b163c62

 I'm not familiar with this repo package, could you please check again for me if this is correct?

Reply:
That's correct, thanks.